docs/reference/airnode/next/chain-idiosyncrasies.md

```diff
@@ -45,7 +45,9 @@ manifests as the `maxFeePerGas` being set to less than the block
 Execution costs on Arbitrum are calculated slightly differently than Ethereum,
 which impacts the gas required to fulfill requests. To account for this, it is
 recommended to use a minimum value of `5000000` for `fulfillmentGasLimit` when
-using both Arbitrum mainnet and testnet. Read more about
+using both Arbitrum mainnet and testnet. Note that `fulfillmentGasLimit` is optional,
+so if you prefer not to specify it, Airnode will attempt to estimate the appropriate
+gas limit automatically. Read more about
 [ArbGas<ExternalLinkImage/>](https://developer.offchainlabs.com/docs/arbgas) gas
 and fees.
 
```

Comment:
Can you delete the commented out text right below this since you've addressed it?